utils/swift_build_support/swift_build_support/products/benchmarks.py

```diff
@@ -85,7 +97,12 @@ def get_dependencies(cls):
                 swiftpm.SwiftPM]
 
 
-def run_build_script_helper(host_target, product, args):
+def _get_toolchain_path(host_target, product, args):
+    """ TODO check if we should prefer using product.install_toolchain_path
```

Comment:
Can you put this in a comment `#` instead of using the docstring

Reply:
Addressed in 0bccd8d2e61